lxd/instances_post.go

```diff
@@ -544,8 +545,9 @@ func createFromCopy(s *state.State, r *http.Request, projectName string, profile
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["instances"] = []string{req.Name, req.Source.Source}
+	resources := map[string][]api.URL{}
+	resources["instances"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", req.Name)}
+	resources["instances_source"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", req.Name, req.Source.Source)}
```

Comment:
Hrm I'm not sure about introducing this new resource category type, as its not really its own resource (and we're making enough changes in this PR as it is).

What was the reason you stopped using the source instance name in "instances" resource URL like it was before?
Was it to align with returning the new instance being created?

Reply:
@tomponline I guess this was motivated by a comment from @monstermunchkin wanting two different URLs to separate `Source.Source` from `resources["instances"]`

Reply:
Hrm, well if we think that the source instance name wasn't appropriate in the "instances" category in this context, then it would be considered a bug, and we can just change it to the new instance name that was created. I don't think we need to introduce a new resource category at this stage.

Reply:
So my thinking was that if we just want to switch to URLs, we'd need 2 separate URLs in `resources["instances"]` as we had 2 entries in `resources["instances"]` before:

```go
resources["instances"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", req.Name), api.NewURL().Path(version.APIVersion, "instances", req.Source.Source)}
```

There's no need for `resources["instances_source"]`.

Reply:
Ah yes that makes sense.

Reply:
ah ok, fixing that